--- runtime.orig/os1_linux.go	2015-12-28 03:47:53.500013399 +0000
+++ runtime/os1_linux.go	2015-12-28 03:49:13.240050327 +0000
@@ -318,9 +318,8 @@
 		return
 	}
 	sa.sa_flags |= _SA_ONSTACK
-	if rt_sigaction(uintptr(i), &sa, nil, unsafe.Sizeof(sa.sa_mask)) != 0 {
-		throw("rt_sigaction failure")
-	}
+	// Terrible hack to mask errors
+	rt_sigaction(uintptr(i), &sa, nil, unsafe.Sizeof(sa.sa_mask))
 }
 
 func getsig(i int32) uintptr {
